### ISSUE TITLE
Doc - First step points to ruslo/hunter/wiki which does not exist

### ISSUE THREAD

**Issue by apivovarov** [cpp-pm/hunter#682]
Doc - First step has the following link to usr.variables#hunter_root. This link is broken. I'm not sure what wiki it points to and where is this wiki now.
https://hunter.readthedocs.io/en/latest/quick-start/boost-components.html
```
First step
Set [HUNTER_ROOT](https://github.com/ruslo/hunter/wiki/usr.variables#hunter_root) environment variable to
```

**Comment by NeroBurner** [comment]
Good catch. It probably should go here: https://hunter.readthedocs.io/en/latest/reference/user-variables.html#hunter-root
